coordinator/main.go

```diff
@@ -85,30 +87,36 @@ func run() (retErr error) {
 	if err != nil {
 		return fmt.Errorf("creating mesh API server: %w", err)
 	}
-	metricsServer := &http.Server{}
+	httpServer := &http.Server{}
+	userapiStarted := false
+	meshapiStarted := false
+
+	startupHandler := probes.StartupHandler{UserapiStarted: &userapiStarted, MeshapiStarted: &meshapiStarted}
```

Comment:
If `StartupHandler` did not have pointer fields, you could do
```suggestion
	startupHandler := probes.StartupHandler{}
	userapiStarted := &startupHandler.UserAPIStarted
	meshapiStarted := &startupHandler.MeshAPIStarted
```